modules/ROOT/pages/rest-api-access.adoc

```diff
@@ -54,7 +68,7 @@ For more on creating Couchbase Server users see the Couchbase Server content her
 
 
 [#lbl-rbac-roles]
-== Authentication Roles
+== RBAC Roles
 
 include::partial$block-caveats.adoc[tags=ee-only-content]
```

Comment:
Again this is more applicable to stuff below
![image](https://user-images.githubusercontent.com/3264971/131556474-f9dfe048-f430-4732-a06a-7042c1777748.png)

![image](https://user-images.githubusercontent.com/3264971/131556679-a736b3aa-b7ca-438c-a93a-b1f6fdf2ed8b.png)

Got some weird overlap.